pkg/payload/v1alpha1/delete.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright © 2018-2019 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
```

Comment:
Let us avoid v1alpha1 folder.
Please refer to my earlier comments.
Please follow this everywhere.

Reply:
yeah, done.